frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableColumnList/TableColumn/TableColumn.module.css

```diff
@@ -48,7 +48,7 @@
   transition: opacity 300ms var(--default-timing-function);
 }
 
-.wrapper:hover .columnType {
```

Comment:
Let me know why this is not working ?👀

Reply:
I didn't know about that either, but I've just been informed(Mr. Miyagi told me). 
You're attempting to access a class named "columnType" from TableNode, 

```
TableNode.module.css

.wrapper:hover .columnType {
  opacity: 1;
}
```

but the real class name is obfuscated. **(it is not ``columnType``)**

<img width="1131" alt="ss 2513" src="https://github.com/user-attachments/assets/b4a2ce8d-24fa-4386-9db4-dffc10b96b94" />




As a result, you couldn't reference the class from another file. I've resolved this by utilizing the data attribute and ``global:.``

Reply:
Thank you for explaining.
`data-*` attributes can be used in that situation, I see 👍

Reply:
Yes😆 This time, we were able to fix the issue by only modifying the CSS.